src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1123,6 +1125,10 @@ private function addSerializerSection(ArrayNodeDefinition $rootNode, callable $e
                         ->arrayNode('default_context')
                             ->normalizeKeys(false)
                             ->useAttributeAsKey('name')
+                            ->beforeNormalization()
```

Comment:
Adding that default in `beforeNormalization` makes it very hard to re-disable the feature because each source will re-enable it in this normalization (well, each source configuring the `default_context`).
This should be done in the `validate` phase instead.

Reply:
see #57803